telegram_phone_number_checker/main.py

```diff
@@ -212,8 +228,22 @@ def main_entrypoint(
     i.e. +491234567891
 
     """
-    client = login(api_id, api_hash, api_phone_number)
-    res = validate_users(client, phone_numbers)
+    asyncio.run(
```

Comment:
Click doesn't support decorating an async function, so it's necessary to use asyncio if the programs entry point is asynchronous.